CHANGELOG.md

```diff
@@ -59,6 +59,12 @@
   Having been [gated](https://github.com/apple/swift/pull/28171) behind a
   compiler warning since at least Swift 5.2, this syntax is now rejected.
 
+* [bugfix][]:
```

Comment:
Nit: This should probably have the PR number in it.

Reply:
https://github.com/apple/swift/pull/71255